CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased](https://github.com/goyek/goyek/compare/v0.6.1...HEAD)
 
+### Added
+
+- Add example for parameter validation in [examples/validation/main.go](examples/validation/main.go).
```

Comment:
I do not think it is worth putting documentation changes in the `CHANGELOG.md`. Can we remove it?

Reply:
I've removed it.